src/app/models/triple-notch-models.model.ts

```diff
@@ -4,6 +4,6 @@
  * @param {string} description - animal description
  */
 export interface Animals {
-  readonly name: string;
```

Comment:
Out of curiosity, why were these removed? Do they need to be changed later? Admittedly I don't totally understand interfaces. Are they just classes that only have variables? Sort of like an object with predetermined keys?